todo_app/data/logs/app.log

```diff
@@ -0,0 +1,111 @@
+### 2022-08-28 19:03:29,645 ### appLog ### INFO ### Title to add: testing_mongo_createCard ###
```

Comment:
Do we want this checked in? I'd suggest maybe not, so it might be a good one to add to the `.gitignore` and to remove the existing file with `git rm --cached todo_app/data/logs/app.log`

Reply:
Something to resolve later in the modules. Git workflow complains if the log file is missing.